talk/owt/sdk/base/peerconnectionchannel.cc

```diff
@@ -98,8 +98,11 @@ void PeerConnectionChannel::AddTransceiver(
 
 const webrtc::SessionDescriptionInterface*
 PeerConnectionChannel::LocalDescription() {
-  RTC_CHECK(peer_connection_);
-  return peer_connection_->local_description();
+  // Unfortunately this isn't entirely safe either as the scoped_refptr isn't
```

Comment:
I don't think this code should change. This is essentially a read-only method on this class and the check makes sense here (like in neighboring methods). the problem is in the calling code. It's calling this method when the peer connection is closed, which points to a race condition. The solution should be in the calling code. See suggestion below.